tests/tests_fabric/test_wrappers.py

```diff
@@ -353,3 +353,36 @@ def test_is_wrapped():
     assert not is_wrapped(dataloader)
     wrapped = _FabricDataLoader(dataloader)
     assert is_wrapped(wrapped)
+
+
+def test_fabric_module_setattr():
```

Comment:
Could you place this test below the test named `test_fabric_module_attribute_lookup`? It would be good to keep them close :)

Reply:
I have changed the location and committed